src/appleseed-max-impl/appleseedoslplugin/oslshaderregistry.cpp

```diff
@@ -546,8 +553,8 @@ void OSLShaderRegistry::add_const_parameter(
         if (osl_param.m_has_default)
         {
             const float r = static_cast<float>(osl_param.m_default_value.at(0));
-            const float g = static_cast<float>(osl_param.m_default_value.at(0));
-            const float b = static_cast<float>(osl_param.m_default_value.at(0));
+            const float g = static_cast<float>(osl_param.m_default_value.at(1));
```

Comment:
That looks like a bug fix? In which case it deserves its own commit.

Reply:
It is bug fix yes. I'll do separate commit for this.

Reply:
@dictoon just the commit or separate PR for this?

Reply:
A separate commit is enough!